go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/argoproj/argo-events v1.4.0
 	github.com/argoproj/argo-workflows/v3 v3.1.6
 	github.com/briandowns/spinner v1.16.0
-	github.com/codefresh-io/go-sdk v0.34.2
+	github.com/codefresh-io/go-sdk v0.34.4
```

Comment:
latest go-sdk is 34.3. 
however, here the cli is set to use 34.2.
in order not to include not-yet-ready changes in 34.3, let's verify